app/views/posts/index.html.erb

```diff
@@ -9,31 +9,34 @@
 <hr class='my-5'>
 
 <% if @posts.any? %>
-  <%= turbo_frame_tag dom_id(@posts.first) do %>
-    <div data-controller="infinite-scroll" data-infinite-scroll-path-value="<%= posts_path(page: @page_number) %>" data-infinite-scroll-target="entry">
-      <% @posts.each do |post| %>
-        <div class='mt-5'>
-          <%= render post %>
+  <div data-controller='scroll_to_top'>
```

Comment:
add an extra div below this line, like
`<div data-scroll-to-top-target='root'></div>`